app/experimenter/nimbus-ui/src/components/SidebarActions/index.scss

```diff
@@ -0,0 +1 @@
+@import "~bootstrap/scss/bootstrap";
```

Comment:
I don't think you should need to import the Bootstrap CSS here, I think that's just done globally